pipelines/ApVerifyWithFakes.yaml

```diff
@@ -126,6 +109,34 @@ tasks:
         config:
             connections.labelName: imageDifferenceNoFakes
             target: imageDifferenceNoFakes:measurement.run
+subsets:
+    apPipe:
+        subset:
+            - isr
+            - characterizeImage
+            - calibrate
+            - imageDifferenceNoFakes
```

Comment:
Where is `imageDifferenceNoFakes` coming from?

Reply:
It's defined higher up in the same file. (And yes, the awkward name is necessary to correctly deal with contracts inherited from `ApVerify.yaml`.)